### ISSUE TITLE
Crowdsourced African History

### ISSUE THREAD

**Issue by Nefert** [MozillaFestival/open-leaders-6#79]
Project Lead: Nefert

Mentor: Abigail Cabunoc Mayes

Welcome to OL6, Cohort C! This issue will be used to track your project and progress during the program. Please use this checklist over the next few weeks as you start Open Leadership Training :tada:.

***

Before Week 1 (Sept 12): Your first mentorship call
- [ ] Complete the OLF self-assessment ([online](https://mzl.la/olf-online-assessment), [printable](https://mzl.la/olf-self-assessment)). If you're a group, each teammate should complete this assessment individually. This is here to help you set your own personal goals during the program. No need to share your results, but be ready to share your thoughts with your mentor.
- [ ] Make sure you know when and how you'll be meeting with your mentor.

Before Week 2 (Sept 19): First Cohort Call (Open by Design)
- [ ] Complete Assignment ['Write Your Vision or Mission Statement'](https://mozilla.github.io/open-leadership-training-series/articles/introduction-to-open-leadership/stating-your-project-vision/). Leave a comment on this issue with your draft vision statement & be ready to share this on the call.
- [ ] Watch ['Mozilla & Working Open'](https://youtu.be/quKdaqlR_9w).
- [ ] Check the [Syllabus](https://mzl.la/ol6-syllabus) for notes and connection info for all the cohort calls.

Before Week 3 (Sept 26): Mentorship call
- [ ] Look up two other projects and comment on their issues with feedback on their vision statement.
- [ ] Complete your Open Canvas ([instructions](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/develop-an-open-project-strategy-with-open-canvas/), [canvas](https://goo.gl/to6PYn)). Comment on this issue with a link to your canvas.
- [ ] [Start your Roadmap](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/start-your-project-roadmap/). Comment on this issue with your draft Roadmap.

Before Week 4 (Oct 3): Cohort Call (Build for Understanding)
- [ ] Look up two other projects and comment on their issues with feedback on their open canvas.
- [ ] [Pick an open license](https://mozilla.github.io/open-leadership-training-series/articles/get-your-project-online/sharing-your-work-in-the-open/) for the work you're doing during the program.
- [ ] Use your canvas to [start writing a README](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/), or landing page, for your project. Link to your README in a comment on this issue.

This issue is here to help you keep track of work during the first month of the program. Please refer to the [OL6 Syllabus](https://mzl.la/ol6-syllabus) for more detailed weekly notes and assignments past week 4.

**Comment by Nefert** [comment]
**Vision Statement Draft**

Working to surface and share as much information about Africa as possible. We hope to build a one stop destination for accurate and objective information on the continent and its cultures that go as far back as possible. 

**Draft 2:** _Working with Afrophiles to unearth and document most of precolonial African experiences, and re-imagine colonial and post-colonial African history through African lens. We hope to build a destination for alternative narratives about the continent and its cultures that Afrophiles and others can use to re-shape their perceptions of themselves and of the continent._

_We are working open because 1)we believe this is a public resource,2) we believe the sheer scale of the project can only be achieved through collective intelligence, and 3)because we wish to develop a version that is not motivated by interests that gain financially and politically by shaping the narrative a certain way._


**Links**
[Open Canvas](https://docs.google.com/presentation/d/1X_VqkbSQaxyiNvX7GRWxxul3tkS2jYEwrizOg94ae4Y/edit?usp=sharing)
[Roadmap](https://github.com/MozillaFestival/open-leaders-6/issues/113)
[ReadMe](https://github.com/MozillaFestival/open-leaders-6/issues/114) 
[Contributing](https://github.com/MozillaFestival/open-leaders-6/issues/117) 
[Code of Conduct](https://github.com/MozillaFestival/open-leaders-6/issues/118)

**Comment by annekainicUSDS** [comment]
This sounds like a really interesting project! I'd be curious to learn more about how it differs from existing tools, like Wikipedia or traditional journalism/academic historical work, to gather and share information about specific topics.

**Comment by skade** [comment]
This projects sounds very, very interesting and I get a good idea of it by reading the statement! It's very ambitious, which is great! Because of this, it could be improved a little, in my opinion, by adding who this is aimed at as a contributor group. Who do you want to work with to make this happen?

**Comment by alzellner** [comment]
@skade I totally agree with you... the "who" is missing for me. I love this project! Citizen history like citizen science, right? Maybe looking at some of those projects might inspire some of the ways you approach your vision.

**Comment by Nefert** [comment]
@annekainicUSDS Thank you! Technically, it doesn't differ much from Wikipedia. It's basically an encyclopedia. Beyond that, there are two  things that make it slightly different; a) It will solely focus on Africa. As you might be aware, contributors from Africa to Wikipedia are very few. This will be a way of rallying others to contribute to a shared knowledge resource by giving them something they can can strongly identify with. 2) Everything will be viewed from the "African perspective". For instance, Johann Ludwig Krapf would the first European to see Mt. Kenya, not the first man.

**Comment by Nefert** [comment]
Thanks @skade & @alzellner! Yes, pretty much like citizen history. I will share an updated vision statement that will include the "who's".

**Comment by Nefert** [comment]
@alzellner I just borrowed our words for the title. Thank you 😊!!

**Comment by Nefert** [comment]
Open Canvas https://docs.google.com/presentation/d/1X_VqkbSQaxyiNvX7GRWxxul3tkS2jYEwrizOg94ae4Y/edit#slide=id.p

**Comment by Nefert** [comment]
@bebatut Here you go! Apologies.
https://docs.google.com/presentation/d/1X_VqkbSQaxyiNvX7GRWxxul3tkS2jYEwrizOg94ae4Y/edit?usp=sharing
